src/schneider_lab_to_nwb/zempolich_2024/zempolich_2024_convert_session.py

```diff
@@ -4,22 +4,46 @@
 import shutil
 from datetime import datetime
 from typing import Optional, Literal
+from pydantic import FilePath, DirectoryPath
 
 from neuroconv.utils import load_dict_from_file, dict_deep_update
 from schneider_lab_to_nwb.zempolich_2024 import Zempolich2024NWBConverter
 
 
 def session_to_nwb(
-    behavior_file_path: str | Path,
-    video_folder_path: str | Path,
-    intrinsic_signal_optical_imaging_folder_path: str | Path,
-    output_dir_path: str | Path,
-    ephys_folder_path: Optional[str | Path] = None,
+    behavior_file_path: FilePath,
```

Comment:
Comment:
I still have not converged myself to be consistent about the folder vs directory usage. 

I am leaning to always use folder as I think directory is far less known and everyone that knows what a directory is knows folder.

Reply:
folder is also shorter, which isn't the most important thing, but it's a nice plus.